src/transformers/models/hubert/modeling_tf_hubert.py

```diff
@@ -418,7 +418,8 @@ def build(self, input_shape):
         if not self.built:
             input_shape = input_shape.as_list()
             # Conv1D output shapes are checked at build time since TF 2.7, so we need to account for padding
-            input_shape[-2] += self.explicit_padding * 2
+            if input_shape[-2] is not None:
+                input_shape[-2] += self.explicit_padding * 2
```

Comment:
Could we add a short comment to explain why this condition? Even a link to a comment in one of your TF PR would be nice.

Reply:
Done!